src/mixer/basetrackplayer.cpp

```diff
@@ -415,8 +416,17 @@ void BaseTrackPlayerImpl::slotLoadFailed(TrackPointer pTrack, const QString& rea
         qDebug() << "Failed to load track (NULL track object)" << reason;
     }
     m_pChannelToCloneFrom = nullptr;
+
     // Alert user.
+    // Show only one message for this track at a time
+    TrackId currTrackId = pTrack->getId();
+    if (pTrack && m_pPrevFailedTrackId == currTrackId) {
+        return;
+    } else if (pTrack) {
+        m_pPrevFailedTrackId = currTrackId;
+    }
     QMessageBox::warning(nullptr, tr("Couldn't load track."), reason);
+    m_pPrevFailedTrackId = TrackId();
```

Comment:
I must admit to not have tested the code, but this line looks like it breaks the new feature.

Reply:
This resets the track id _after the message was closed_ so there can be only one message for a certain track id **at a time**, in case this slot is called repeatedly.
"at a time" is the point here, I want to avoid multiple identical stacked messages.
Of course we must allow to load the failed again over and over again and show a new message then.

So: No, this _enables_ the new feature : )